packaging/make-git-snapshot.sh

```diff
@@ -29,4 +29,15 @@ ls -al ${TARFILE_TMP}
     tar -A -f ${TARFILE_TMP} submodule.tar
     rm submodule.tar
 done
+tmpd=$(mktemp -d)
+touch ${tmpd}/.tmp
+rm -f rust.tar
+(cd ${tmpd}
+ mkdir -p .cargo vendor
+ cargo vendor -q --sync ${TOP}/rust/Cargo.toml vendor
+ cp ${TOP}/rust/Cargo.lock .
+ cp ${TOP}/rust/cargo-vendor-config .cargo/config
+ tar --transform='s,^,rust/,' -vrf ${TARFILE_TMP} .
```

Comment:
Neat!

Reply:
Though let's drop the `-v`? Seems inconsistent to be printing these when we're not printing anything else during `dist-snapshot`.